### PR TITLE
tests: run on 2.0 on CI

### PR DESCRIPTION
**Pull request by robertkowalski**
this makes it possible to test new 2.0 features like mango or and
all other changes related to the 2.0 release.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,26 +1,37 @@
-language: node_js
-node_js:
-  - '0.10'
+language: erlang
+otp_release:
+   - 17.3
```
Comment (kxepal) [inline .travis.yml]: Travis dropped 17.3 support in favour of 17.4

**Comment by robertkowalski** [comment]
thx fixed.

**Comment by garrensmith** [comment]
+1 this is very cool

**Comment by robertkowalski** [comment]
merged as

38544abe7df4bf042b0a79bdb74cf343c97469d8
6bf2ebf62166485745690a07c2797ece9c25f553
db20167e425b41a0370e54cf61cca7042268358d
592638a108c0bfda94f84428a73cafd891bbb979

**Pull request closed without merging**